include/tvm/ir/expr.h

```diff
@@ -161,7 +163,8 @@ class RelayExprNode : public BaseExprNode {
   template<typename TTypeNode>
   inline const TTypeNode* type_as() const;
 
-  static constexpr const char* _type_key = "relay.Expr";
+  static constexpr const char* _type_key = "RelayExpr";
```

Comment:
I'm not sure I like this change, its better to have the keys name spaced to Relay imo.

Reply:
This change is mainly for the Base class(given the c++ name is RelayExpr under the root). The relay sub-classes are under the relay namespace.